assets/tabs.css

```diff
@@ -0,0 +1,64 @@
+.tabs__media--small {
+  height: 19.4rem;
+}
+
+.tabs__media--medium {
+  height: 43.5rem;
+}
+
+@media screen and (min-width: 750px) {
+  .tabs__media--small {
+    height: 31.4rem;
+  }
+
+  .tabs__media--medium {
+    height: 69.5rem;
+  }
+}
+
+@media screen and (max-width: 749px) {
+  .tabs__grid {
+    flex-direction: column-reverse;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .tabs__grid--reverse {
+    flex-direction: row-reverse;
+  }
+}
+
+.color:not(.color-background-1).accordion,
+.color:not(.color-background-1).tabs-wrapper {
+  padding-left: 1.5rem;
+}
+
+.color:not(.color-background-1).accordion:not(:first-child) {
+  margin-top: 1rem;
+}
+
+.tabs.color:not(.color-background-1),
+.color:not(.color-background-1).tabs-wrapper {
+  padding-bottom: 5rem;
+  padding-bottom: 3rem;
+}
+
+@media screen and (min-width: 750px) {
+  .tabs.color:not(.color-background-1),
+  .color:not(.color-background-1).tabs-wrapper {
+    padding-top: 7rem;
+    padding-bottom: 9rem;
+  }
+}
+
+.caption-with-letter-spacing + h2 {
+  margin-top: 1rem;
+}
+
+.tabs .accordion:hover {
```

Comment:
@Oliviammarcello Should the hover state be applied to the product accordion as well?

Reply:
Yes please!

Reply:
Created an issue here to update the block on the product page: https://github.com/Shopify/dawn/issues/1001